### ISSUE TITLE
Helm 3: Does not allow charts with resources in multiple namespaces

### ISSUE THREAD

**Issue by limscoder** [helm/helm#5953]
Output of `helm version`:

v3.0.0-alpha.1

Helm errors when the chart contains a resource in a different namespace than the current kube-context, even when the user has cluster-admin access. 

```
the namespace from the provided object \"target\" does not match the namespace \"default\". You must pass '--namespace=target' to perform this operation.
```

This prevents charts with resources in multiple namespaces from being installed.

**Comment by hickeyma** [comment]
@limscoder https://github.com/helm/helm/issues/5628 is opened which covers not being able to specify namespace instead of using the user kube context namespace.

WRT multi-namespaced charts, @bacongobbler @adamreese are they on  the agenda for Helm 3.

**Comment by loganmzz** [comment]
It makes impossible to have chart which create rolebinding to external namespace (and more specially against system roles).

**Comment by bacongobbler** [comment]
#6046 implements a fix for this. Please feel free to give that a shot. I've tested this against several charts and I'm fairly confident this fixes that particular issue.

**Comment by bacongobbler** [comment]
closed via #6046
